doc/design/request_response_communication.md

```diff
@@ -110,6 +110,21 @@ A client can ask a server to block if its response queue is full.
 The server will obey if the corresponding `clientTooSlowPolicy` is set to `ConsumerTooSlowPolicy::WAIT_FOR_CONSUMER`.
 If the options don't match, the client will not be connected to the server, similar to the behavior of publisher and subscriber.
 
+#### Client Service Discovery
```

Comment:
To view this with rendered diagrams use the three dots on the right and select `View file`